Makefile

```diff
@@ -12,3 +12,27 @@ build:
 		-X github.com/ricoberger/jaeger-exporter/pkg/version.BuildUser=${BUILDUSER} \
 		-X github.com/ricoberger/jaeger-exporter/pkg/version.BuildDate=${BUILDTIME}" \
 		-o ./bin/exporter ./cmd/exporter;
+
```

Comment:
Can you please keep this file aligned with the configuration from my other repositories, e.g. https://github.com/ricoberger/echoserver/blob/main/Makefile